### PR TITLE
Change return types to CInt

### PR DESCRIPTION
**Pull request by ekrich**

### CODE REVIEW

File: nativelib/src/main/scala/scala/scalanative/posix/unistd.scala [hunk 1]
```diff
@@ -16,8 +16,8 @@ object unistd {
 
   type off_t = CLongLong
 
-  def sleep(seconds: CUnsignedInt): Int                           = extern
-  def usleep(usecs: CUnsignedInt): Int                            = extern
+  def sleep(seconds: CUnsignedInt): CInt                          = extern
```
Comment (densh) [inline nativelib/src/main/scala/scala/scalanative/posix/unistd.scala]: Actually, this one is `unsigned int sleep(unsigned int);` in C, therefore it should return `CUnsignedInt`.

Comment (ekrich) [inline nativelib/src/main/scala/scala/scalanative/posix/unistd.scala]: Depends on https://github.com/scala-native/scala-native/issues/693

Comment (densh) [inline nativelib/src/main/scala/scala/scalanative/posix/unistd.scala]: I see, lets use `CInt` for now. And as soon as #693 is fixed we'll update the bindings.

Comment (ekrich) [inline nativelib/src/main/scala/scala/scalanative/posix/unistd.scala]: Ok, should be good to go now.

**Comment by densh** [comment]
LGTM!

**Pull request merged**